src/control/server/config.go

```diff
@@ -40,17 +41,12 @@ import (
 )
 
 const (
-	defaultRuntimeDir        = "/var/run/daos_server"
-	defaultConfigPath        = "../etc/daos_server.yml"
-	defaultSystemName        = "daos_server"
-	defaultPort              = 10001
-	configOut                = ".daos_server.active.yml"
-	relConfExamplesPath      = "../utils/config/examples/"
-	msgBadConfig             = "insufficient config file, see examples in "
-	msgConfigNoProvider      = "provider not specified in config"
-	msgConfigNoPath          = "no config path set"
-	msgConfigNoServers       = "no servers specified in config"
-	msgConfigBadAccessPoints = "only a single access point is currently supported"
+	defaultRuntimeDir   = "/var/run/daos_server"
+	defaultConfigPath   = "../etc/daos_server.yml"
+	defaultSystemName   = "daos_server"
+	defaultPort         = 10001
```

Comment:
I think we should consider defining this in a central location (common, I guess) as DefaultControlPort. Then we only need to update it once if it gets changed again.